kolibri/utils/cli.py

```diff
@@ -14,9 +14,9 @@
 
 import kolibri  # noqa
 from kolibri import dist as kolibri_dist  # noqa
-sys.path = [
+sys.path = sys.path + [
     os.path.realpath(os.path.dirname(kolibri_dist.__file__))
-] + sys.path
+]
```

Comment:
This is a major change with unintended repercussions. it's usually better to have standard library be the first priority, but other installed libs are now getting imported first vs. what's in kolibri.dist.

Note that this change won't affect pex files (our main installation method), but will affect whl files.